Reflection_Engine/Query/IsAssignableFromIncludeGenerics.cs

```diff
@@ -56,6 +56,26 @@ public static bool IsAssignableFromIncludeGenerics(this Type assignableTo, Type
         }
 
         /***************************************************/
+
+        [Description("Checks if a type is assignable from another type by first checking the system IsAssignableFrom." +
+            "\nIf this is false, checks if the assignable is generic and tests if it can be assigned as a generics version." +
+            "\nIf this is false, checks if the types are the same type, only a byRef version (&).")]
+        [Input("assignableTo", "The type to check if it can be assigned to.")]
+        [Input("assignableFrom", "The type to check if it can be assigned from.")]
+        [Output("result", "Returns true if 'assignableTo' is assignable from 'assignableFrom'.")]
+        public static bool IsAssignableFromIncludeGenericsAndRefTypes(this Type assignableTo, Type assignableFrom)
+        {
+            if (IsAssignableFromIncludeGenerics(assignableTo, assignableFrom))
```

Comment:
This method looks new and probably needs some `null-handling`

Reply:
Thanks, done 6a77ecb488d2af5ca5561d062af5c28b09c94293